<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSaveScenarioDialog</class>
 <widget class="QDialog" name="CSaveScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Scenario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Current Scenario</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ScenarioInfo">
     <property name="title">
      <string>Scenario Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Name">
        <property name="text">
         <string>Scenario Name:*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Name">
        <property name="placeholderText">
         <string>Enter scenario name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Description">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="textEdit_Description">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Enter optional description</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_SaveLocation">
     <property name="title">
      <string>Save Location</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_SaveLocation">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_BrowseLocation">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #9E9E9E,
                                stop:0.5 #8D8D8D,
                                stop:1 #757575);
    border: 2px solid rgba(158, 158, 158, 0.3);
    color: white;
    padding: 8px 16px;
    border-radius: 8px;
    font-weight: 600;
    letter-spacing: 0.5px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #BDBDBD,
                                stop:0.5 #A8A8A8,
                                stop:1 #9E9E9E);
    border: 2px solid rgba(158, 158, 158, 0.5);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #616161,
                                stop:1 #424242);
    border: 2px solid rgba(97, 97, 97, 0.8);
    padding: 9px 16px 7px 16px;
}</string>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Status">
     <property name="text">
      <string>Please enter a scenario name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2196F3,
                                stop:0.5 #1E88E5,
                                stop:1 #1976D2);
    border: 2px solid rgba(33, 150, 243, 0.3);
    color: white;
    padding: 8px 16px;
    border-radius: 8px;
    font-weight: 600;
    letter-spacing: 0.5px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #42A5F5,
                                stop:0.5 #2196F3,
                                stop:1 #1E88E5);
    border: 2px solid rgba(33, 150, 243, 0.5);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #1565C0,
                                stop:1 #0D47A1);
    border: 2px solid rgba(13, 71, 161, 0.8);
    padding: 9px 16px 7px 16px;
}
QPushButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #BDBDBD,
                                stop:1 #9E9E9E);
    border: 2px solid rgba(189, 189, 189, 0.3);
    color: rgba(255, 255, 255, 0.5);
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #9E9E9E,
                                stop:0.5 #8D8D8D,
                                stop:1 #757575);
    border: 2px solid rgba(158, 158, 158, 0.3);
    color: white;
    padding: 8px 16px;
    border-radius: 8px;
    font-weight: 600;
    letter-spacing: 0.5px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #BDBDBD,
                                stop:0.5 #A8A8A8,
                                stop:1 #9E9E9E);
    border: 2px solid rgba(158, 158, 158, 0.5);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #616161,
                                stop:1 #424242);
    border: 2px solid rgba(97, 97, 97, 0.8);
    padding: 9px 16px 7px 16px;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
